### ISSUE TITLE
x/build: gotip-ios-amd64 and gotip-ios-arm64 SlowBots always fail

### ISSUE THREAD

**Issue by FiloSottile** [golang/go#66164]
See https://cr-buildbucket.appspot.com/build/8754129201848019169 and https://cr-buildbucket.appspot.com/build/8754129201848019153 from https://go.dev/cl/569655, which would really need testing on iOS.

**Comment by prattmic** [comment]
iOS builders haven't yet been added to LUCI, see the known issue dashboard: https://ci.chromium.org/p/golang/g/go-gotip-known_issue/console

So you'll need to use the old slow bots (same as usual, TRY= and Run-TryBots+1).

Unfortunately, that iOS builder has been continuously failing for months (#61446). It tends to make it through the crypto packages though, so it may be good enough for your needs...

(I think we also wanted to remove these builders from the "choose tryjobs" UI to make this less confusing. @mknyszek)

**Comment by mknyszek** [comment]
Yeah, I have a TODO to move the "known issue" builders out of that list. Sorry about the delay there.
